app/cli/cmd/workflow_workflow_run_describe.go

```diff
@@ -217,7 +220,18 @@ func encodeAttestationOutput(run *action.WorkflowRunItemFull) error {
 		return encodeJSON(run.Attestation.Statement())
 	case formatAttestation:
 		return encodeJSON(run.Attestation.Envelope)
+	case formatPayloadPAE:
+		return describePayload(run, writer)
 	default:
 		return ErrOutputFormatNotImplemented
 	}
 }
+
+func describePayload(run *action.WorkflowRunItemFull, writer io.Writer) error {
```

Comment:
this name can be a little but confusing to me, why is it called `describePayload`?

Reply:
I agree, I'll change it.